debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CassandraConnectorConfig.java

```diff
@@ -461,16 +447,31 @@ public Converter getValueConverter() throws CassandraConnectorConfigException {
     }
 
     private Map<String, Object> keyValueConverterConfigs(String converterPrefix) {
-        return configs.entrySet().stream()
+        return this.getConfig().asMap().entrySet().stream()
                 .filter(entry -> entry.toString().startsWith(converterPrefix))
-                .collect(Collectors.toMap(entry -> entry.getKey().toString().replace(converterPrefix, ""), entry -> entry.getValue()));
+                .collect(Collectors.toMap(entry -> entry.getKey().replace(converterPrefix, ""), entry -> entry.getValue()));
     }
 
     @Override
     public String toString() {
-        return configs.entrySet().stream()
-                .filter(e -> !e.getKey().toString().toLowerCase().contains("username") && !e.getKey().toString().toLowerCase().contains("password"))
+        return this.getConfig().asMap().entrySet().stream()
+                .filter(e -> !e.getKey().toLowerCase().contains("username") && !e.getKey().toLowerCase().contains("password"))
                 .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue))
                 .toString();
     }
+
+    @Override
+    public String getContextName() {
+        return Module.contextName();
+    }
+
+    @Override
+    protected SourceInfoStructMaker<? extends AbstractSourceInfo> getSourceInfoStructMaker(Version version) {
+        switch (version) {
```

Comment:
This shouldn't be needed, or do we actually want to maintain two different versions of the source struct? I don't think so, given the connector's "incubating" state. The notion of the "legacy" struct maker exists for the other connectors, as we did some technically non-backwards compatible changes to `source` in 0.10 (for the sake of general consistency and stream-lining, this might btw. relevant for you when upgrading your MySQL connectors) and we wanted to have a transitioning period where the old one still is available, too. This shouldn't be needed for Cassandra.

Reply:
> This shouldn't be needed, or do we actually want to maintain two different versions of the source struct? I don't think so, given the connector's "incubating" state. The notion of the "legacy" struct maker exists for the other connectors, as we did some technically non-backwards compatible changes to `source` in 0.10 (for the sake of general consistency and stream-lining, this might btw. relevant for you when upgrading your MySQL connectors) and we wanted to have a transitioning period where the old one still is available, too. This shouldn't be needed for Cassandra.

Understood, updated accordingly!